indico/modules/designer/placeholders.py

```diff
@@ -268,6 +268,12 @@ def render(cls, registration):
         return format_currency(registration.price, registration.currency, locale='en_GB')
 
 
+class RegistrationIdPlaceholder(RegistrationPlaceholder):
+    name = 'registrationid'
```

Comment:
`registration_id` or `registration_friendly_id` (if you go for the latter, also update the class name)

Reply:
Done